xml/schema/CRM/Certificate/CompuCertificateTemplateImageFormat.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="iso-8859-1" ?>
+
+<table>
+  <base>CRM/Certificate</base>
+  <class>CompuCertificateTemplateImageFormat</class>
+  <name>civicrm_compu_certificate_template_image_format</name>
```

Comment:
usually CiviCRM core tables start with `civicrm_` , so I prefer to remove such prefix from extension tables so it is clear for anyone who is looking into the database where that table is coming from, so maybe you can go with: `compucertificate_template_image_format` instead.